File: 代码/kyzh_parent/kyzh_manager_web/src/main/webapp/js/service/articleService.js
//服务层
app.service('articleService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../article/findAll.do');		
	}
	//分页 
	this.findPage=function(pageNo,pageSize,searchEntity){
		return $http.post('../article/findPage.do?pageNo='+pageNo+'&pageSize='+pageSize, searchEntity);
		//return $http.post('../testdata.json'); //前端测试效果的json文件
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../article/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../article/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../article/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../article/delete.do?ids='+ids);
	}
	//查询文章分类下拉列表
    this.findBrandList = function () {
        return $http.get('../dataSorting/selectOptions.do');
    }
});

File: 代码/kyzh_parent/kyzh_article_interface/src/main/java/com/kyzh/article/service/ArticleService.java
package com.kyzh.article.service;
import java.util.List;
import com.kyzh.pojo.TbArticle;

import entity.Article;
import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ArticleService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbArticle> findAll();
	
	
	/**
	 * 增加
	*/
	public void add(Article article);
	
	
	/**
	 * 修改
	 */
	public void update(Article article);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Article findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页+模糊搜索
	 * 参数一模糊搜索条件
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findAll(TbArticle article, int pageNum, int pageSize);
	
}
